### PR TITLE
feat: Support Nuxt 3

### PR DESCRIPTION
**Pull request by wattanx**
https://github.com/nuxt-community/style-resources-module/issues/213

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Support Nuxt 3 with reference to https://github.com/wattanx/style-resources-module.
(If you would prefer to make the PR a little smaller in size, please let me know.)

I think `Same Repositories, same package name` is better.

### CODE REVIEW

**Comment by davestewart** [comment]
👀

**Comment by manniL** [comment]
In this case, same package + same repository is more than fine 👍

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
-# Style Resources for Nuxt 2 - Nobody likes extra @import statements!
+# Style Resources for Nuxt 3 - Nobody likes extra @import statements!
 
 [![npm version][npm-version-src]][npm-version-href]
 [![npm downloads][npm-downloads-src]][npm-downloads-href]
 [![Codecov][codecov-src]][codecov-href]
 [![License][license-src]][license-href]
 
->
+If you would like to use the stable version for Nuxt 2, please see the [`mater` branch](https://github.com/nuxt-community/style-resources-module/tree/master)
```
Comment (manniL) [inline README.md:8]:
```suggestion
If you would like to use the stable version for Nuxt 2, please see the [`master` branch](https://github.com/nuxt-community/style-resources-module/tree/master)
```

Comment (manniL) [reply README.md:8]: Also, we might want to say "use major version XYZ" here? 🤔

File: README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+
+If you are using vite, this module is unnecessary.
```
Comment (manniL) [inline README.md:10]:
```suggestion
If you are using Vite, this module is not needed.
```

File: package.json [hunk 3]
```diff
@@ -8,7 +8,15 @@
       "name": "Alexander Lichter <npm@lichter.io>"
```
Comment (manniL) [inline package.json:8]: Please add yourself here too! 👍

**Review by manniL — APPROVED** [review]
LGTM besides a few nits 👍

File: README.md (hunk 1 above)
Comment (wattanx) [reply README.md]: Added description to use `nuxtjs/style-resources@^1`.
Is it OK to create a `v1` branch?
(If OK, I would like you to make it as I do not have the authorization to do so 🙏 .)

https://github.com/nuxt-community/style-resources-module/pull/214/commits/ae115a1385f83c8489adf1ebb513e20c07b9bc40

Comment (wattanx) [reply README.md]: I created `v1` branch.
https://github.com/nuxt-community/style-resources-module/tree/v1

**Comment by harlan-zw** [comment]
Not sure what this Travis CI integration is doing. Going to merge as it seems good to go, can make any fixes outside of this PR.

**Comment by manniL** [comment]
@danielroe could you remove travis from the org? I sadly don't have access to it 🙈

**Pull request merged**